packages/server/lib/config_options.ts

```diff
@@ -41,6 +41,11 @@ export const options = [
     defaultValue: 'cypress/component',
     validation: v.isStringOrFalse,
     isFolder: true,
+  }, {
+    name: 'component',
+    // runner-ct overrides
+    defaultValue: {},
+    validation: v.isPlainObject,
```

Comment:
A very minor nitpick, but `component` should go before `componentFolder` alphabetically 😬